test/mailinSpec.js

```diff
@@ -16,15 +16,34 @@ chai.use(chaiAsPromised);
 var should = null;
 should = chai.Should();
 
+var server = express(),
```

Comment:
I had to move this up here, to make it listen before `Mailin.start()` gets called and checks `/webhook`.